app/src/main/java/com/boolder/boolder/view/map/MapFragment.kt

```diff
@@ -159,6 +160,7 @@ class MapFragment : Fragment(), BoolderMapListener {
             onSelectProblemOnMap = { problemId ->
                 binding.mapView.selectProblem(problemId)
                 mapViewModel.updateCircuitControlsForProblem(problemId)
+                bottomSheetBehavior.state = STATE_EXPANDED
```

Comment:
This line should not be necessary, as `onTopoUnselected()` from the view model will reset the topo state, allowing the topo bottom sheet to be expanded in the usual way